cmake/Modules/FindQwt.cmake

```diff
@@ -29,8 +30,9 @@ find_path(QWT_INCLUDE_DIRS
 )
 
 find_library (QWT_LIBRARIES
-  NAMES ${PC_QWT_LIBRARIES} qwt6-${QWT_QT_VERSION} qwt-${QWT_QT_VERSION}
+  NAMES ${PC_QWT_LIBRARIES} qwt6 qwt6-${QWT_QT_VERSION} qwt-${QWT_QT_VERSION}
```

Comment:
We removed that on purpuose, until all the qwt6 libraries linked to qt4 faded away. You should be able to override settings here with a cmake switch if you know where your qwt library is.